Dockerfile

```
@@ -1,7 +1,11 @@
 ARG PYTHON_VERSION=2.7
-
 FROM python:${PYTHON_VERSION}
 
+# Add SSH keys and set permissions
+COPY tests/ssh-keys /root/.ssh
+RUN chmod 600 /root/.ssh/id_rsa && \
+    chmod 600 /root/.ssh/id_rsa.pub
```

I think with engine 20.x, we'll be able to use `COPY --chmod` https://github.com/moby/buildkit/pull/1492

would a `chmod -R` work for this?

```suggestion
RUN chmod -R 600 /root/.ssh/
```